### ISSUE TITLE
On some machines, quickFilters is not loading at startup

### ISSUE THREAD

**Issue by RealRaven2000** [RealRaven2000/quickFilters#66]
On some installations quickFilters doesn't load any UI on startup, and needs to be reloaded from Add-ons Manager to work.

A similar problem was found in QuickFolders following the execution of accounts code on startup, with a similar error message as thisL:

```
jar:file:///C:/Users/Stephen/AppData/Roaming/Thunderbird/Profiles/tolc144y.default/extensions/quickfolders@curious.be.xpi!/chrome/content/api/
ex_accounts/implementation.js :: traverseSubfolders :: line 142"  data: no] 
implementation.js:142:25
Uncaught (in promise) Error: An unexpected error occurred undefined

```

**Comment by RealRaven2000** [comment]
The following test version should guard against the problem in Thunderbird startup [until the bug is fixed](https://bugzilla.mozilla.org/show_bug.cgi?id=1719739) in Thunderbird:

[quickFilters-wx-5.3pre74.zip](https://github.com/RealRaven2000/quickFilters/files/6797268/quickFilters-wx-5.3pre74.zip)

To install, download the zip file and then drag it into Thunderbird Add-ons Manager.

It will log in the Error console which of the accounts gives trouble when trying to access it with the accounts API.
Either:
traverseSubfolders failed: account ex
convertAccount failed: account ex

(Clicking on account will expand the object and show which account gave problems on startup)

**Comment by RealRaven2000** [comment]
Fixed in 5.3, released on 04/04/2021
